### ISSUE TITLE
Remove `/pb/google/protobuf/Timestamp.proto`

### ISSUE THREAD

**Issue by mic-max** [open-telemetry/opentelemetry-demo#107]
# Feature Request

This was added as a hacky way to get around protobuf compilation errors while migrating from keeping copies of `demo.proto` in the repository along with the language specific files/classes generated from the aforementioned proto file.

Installing `protoc` using `apk` might not be setting the include path correctly to this directory, which should be included with the install: https://github.com/protocolbuffers/protobuf/tree/main/src/google

The only needed file was `Timestamp.proto` so that's the only one we copied over.

Consider

- https://github.com/znly/docker-protobuf
- https://medium.com/windmill-engineering/how-to-compile-protobufs-when-you-have-friends-who-want-to-use-them-too-b76d7fdc247e
- https://stackoverflow.com/questions/56031098/protobuf-timestamp-not-found

**Comment by cartersocha** [comment]
Is this still needed? Closing for now

**Comment by mic-max** [comment]
Yes. Ideally this file: https://github.com/open-telemetry/opentelemetry-demo/blob/main/pb/google/protobuf/timestamp.proto should not be part of this repository.
